demos/rogers/experiment.py

```diff
@@ -1,25 +1,31 @@
 """Replicate Rogers' paradox by simulating evolution with people."""
 
 from dallinger.experiments import Experiment
-from dallinger.information import Gene, Meme, State
-from dallinger.nodes import Source, Agent, Environment
+from dallinger.information import Meme
+from dallinger.nodes import Agent, Environment
 from dallinger.networks import DiscreteGenerational
 from dallinger.models import Node, Network, Participant
-from dallinger import transformations
-from sqlalchemy import Integer, Float
-from sqlalchemy.ext.hybrid import hybrid_property
-from sqlalchemy.sql.expression import cast, false
+from sqlalchemy.sql.expression import false
 from sqlalchemy import and_
-from operator import attrgetter
 import random
 
 
 class RogersExperiment(Experiment):
     """The experiment class."""
 
     def __init__(self, session):
-        """Create the experiment."""
+        """Call the same function in the super (see experiments.py in dallinger).
+
+        The sources module is imported here because it must be imported at
+        runtime.
+
+        A few properties are then overwritten.
+
+        Finally, setup() is called.
+        """
         super(RogersExperiment, self).__init__(session)
+        import sources
+        self.sources = sources
```

Comment:
(The same comment I left on the MCMCP demo applies here. With so many custom objects, renaming to something more generic is probably the way to go.)